python/ray/services.py

```diff
@@ -301,12 +301,10 @@ def _compute_version_info():
         A tuple containing the version information.
     """
     ray_version = ray.__version__
-    ray_location = os.path.abspath(ray.__file__)
     python_version = ".".join(map(str, sys.version_info[:3]))
     cloudpickle_version = cloudpickle.__version__
     pyarrow_version = pyarrow.__version__
-    return (ray_version, ray_location, python_version, cloudpickle_version,
-            pyarrow_version)
+    return (ray_version, python_version, cloudpickle_version, pyarrow_version)
```

Comment:
I removed the `ray_location` because really it shouldn't matter where Ray is installed... that check would have just caused a bunch of problems I think.